GitHubCore.ps1

```diff
@@ -154,7 +151,9 @@ function Invoke-GHRestMethod
     # we'll just always continue the existing one...
     $stopwatch.Start()
 
-    $url = "$script:gitHubApiUrl/$UriFragment"
+    $hostName = $(Get-GitHubConfiguration -Name "ApiHostName")
+
+    $url = "https://api.$hostname/$UriFragment"
```

Comment:
nit: Can you use the same casing as the variable declaration?

```suggestion
    $url = "https://api.$hostName/$UriFragment"
```

Reply:
This was in fact a typo, nice catch 👀